src/django_aetos/views.py

```diff
@@ -18,12 +19,45 @@ def export_metrics(request):
                 metric["add_help"] = True
             known_metrics.add(metric["name_without_labels"])
             metrics.append(metric)
+    
+    return metrics
+
+def check_auth(request):
+    ip_address = request.META['REMOTE_ADDR']
+    allowed_ips = app_settings.IP_ALLOWLIST
+    
+    if app_settings.ENABLE_IP_ALLOWLIST and ip_address in allowed_ips:
+        return True
+    elif not app_settings.ENABLE_IP_ALLOWLIST:
+        return True
+    else:
+        return False
 
-    response = render(
-        request,
-        "metrics/export.txt",
-        context={"metrics": metrics},
-        content_type="text/plain",
-    )
-    response.content = re.sub(b"\n+", b"\n", response.content)
-    return response
+def check_ips(request):
+    try:
+        authorization_header = request.META['HTTP_AUTHORIZATION']
+    except KeyError:
+        authorization_header = ""
+    type, sep, token = authorization_header.partition(' ')
+    allowed_tokens = app_settings.AUTH_TOKENLIST
+    
+    if app_settings.ENABLE_AUTH and type == "Bearer" and token in allowed_tokens:
```

Comment:
probably use `is True` here so we don't interpret true-ish values from wrong configs as true.

Reply:
Fixed in 1ddd9c87f8bb9c78f7dc3998e406cf8b562178f4